README.md

```diff
@@ -37,8 +37,18 @@ To run the app locally, use:
 bin/dev
 ```
 
-You should now be able to access the application at [http://localhost:3000](http://localhost:3000).
+## Subdomain Setup
```

Comment:
Does this make sense to be put below the "Accessing Roles" section, I think that might make sense to appear first?

Reply:
moved